apps/prairielearn/package.json

```diff
@@ -141,26 +141,26 @@
     "object-hash": "^3.0.0",
     "on-finished": "^2.4.1",
     "on-headers": "^1.0.2",
-    "openai": "^4.82.0",
+    "openai": "^4.86.1",
     "openid-client": "^5.7.1",
     "parse5": "^7.2.1",
     "passport": "^0.7.0",
     "passport-azure-ad": "^4.3.5",
     "pem": "^1.14.8",
-    "pg": "^8.13.1",
+    "pg": "^8.13.3",
     "plist": "^3.1.0",
     "popper.js": "^1.16.1",
-    "preact": "^10.25.4",
-    "prettier": "3.4.2",
+    "preact": "^10.26.4",
+    "prettier": "3.5.2",
     "qrcode-svg": "^1.1.0",
     "qs": "^6.14.0",
-    "quill": "2.0.2",
+    "quill": "2.0.3",
```

Comment:
#11281 
```suggestion
    "quill": "2.0.2",
```

Reply:
Thanks, fixed in 812046b. Yet again wishing Yarn had a "no really, please pin this thing" option so that I couldn't screw this up when running `yarn upgrade-interactive`.